docs/v2.4/quick-start/demo-node.md

```diff
@@ -2,294 +2,7 @@
-   Login successful.
-   ```
-
-   If you are in an environment where Telepresence cannot launch a
-   local browser for you to interact with, you will need to pass the
-   [`--apikey` flag to `telepresence
-   login`](../../reference/client/login/).
-
-3. Start the intercept again:
-`telepresence intercept dataprocessingservice --port 3000`
-
-   You will be asked for your ingress layer 3 address; specify the front end service: `verylargejavaservice.default`
-   Then when asked for the port, type `8080`, for "use TLS", type `n`.  The default for the fourth value is correct so hit enter to accept it
-
-  ```
-  $ telepresence intercept dataprocessingservice --port 3000
-
-    To create a preview URL, telepresence needs to know how cluster
-    ingress works for this service.  Please Select the ingress to use.
-
-    1/4: What's your ingress' layer 3 (IP) address?
-         You may use an IP address or a DNS name (this is usually a
-         "service.namespace" DNS name).
-
-           [no default]: verylargejavaservice.default
-
-    2/4: What's your ingress' layer 4 address (TCP port number)?
-
-           [no default]: 8080
-
-    3/4: Does that TCP port on your ingress use TLS (as opposed to cleartext)?
-
-           [default: n]: n
-
-    4/4: If required by your ingress, specify a different layer 5 hostname
-         (TLS-SNI, HTTP "Host" header) to access this service.
-
-           [default: verylargejavaservice.default]:
-
-    Using deployment dataprocessingservice
-    intercepted
-        Intercept name  : dataprocessingservice
-        State           : ACTIVE
-        Destination     : 127.0.0.1:3000
-        Intercepting    : HTTP requests that match all of:
-          header("x-telepresence-intercept-id") ~= regexp("86cb4a70-c7e1-1138-89c2-d8fed7a46cae:dataprocessingservice")
-        Preview URL     : https://<random-subdomain>.preview.edgestack.me
-        Layer 5 Hostname: verylargejavaservice.default
-  ```
-
-4. Wait a moment for the intercept to start; it will also output a preview URL.  Go to this URL in your browser, it will be the <strong style="color:orange">orange</strong> version of the app.
-
-5. Now go again to [http://verylargejavaservice:8080](http://verylargejavaservice:8080), it’s still <strong style="color:green">green</strong>.
-
-Normal traffic coming to your app gets the <strong style="color:green">green</strong> cluster service, but traffic coming from the preview URL goes to your laptop and gets the <strong style="color:orange">orange</strong> local service!
-
-<Alert severity="success">
-  The <strong>Preview URL</strong> now shows exactly what is running on your local laptop -- in a way that can be securely shared with anyone you work with.
-</Alert>
-
-## <img class="os-logo" src="../../images/logo.png"/> What's Next?
-
-<QSCards/>
+You can find a guide to getting started with Telepresence on [here](https://www.getambassador.io/docs/telepresence/2.4/quick-start/demo-node/).
```

Comment:
Perhaps word it something like: "To use one of our demo clusters to learn how Telepresence can be used  to start intercepting services to speed up local and development, you can find a guide [here]"